nilearn/plotting/tests/test_view_stat_map_change_warning.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+
+from nilearn.plotting.html_stat_map import (_view_stat_map_change_warning,
+                                            view_stat_map)
+from nilearn._utils.testing import generate_fake_fmri
```

Comment:
This has indeed be moved to nilearn._utils.data_gen

Reply:
Hmmm, not in my local copy when I wrote that, why was this failing in the CI... will see.